src/exec/interface.rs

```diff
@@ -58,7 +58,7 @@ impl RunOptions for Context {
     }
 
     fn pid(&self) -> i32 {
-        self.process.pid
+        self.process.pid.get()
```

Comment:
The pid function should probably return a ProcessId rather than unwrapping the ProcessId into an i32 and then wrapping it again at every call site.